api/tests/unit/domain/models/Skill_test.js

```diff
@@ -1,4 +1,4 @@
-import { beforeEach, describe, expect, it, vi } from 'vitest';
+import { beforeEach, describe, describe as context, expect, it, vi } from 'vitest';
```

Comment:
Pourquoi ne pas utiliser "describe" ?

Reply:
pour que ça ressemble à mocha.
J'ai une habitude mentale d'avoir le describe qui contient le module testé.
Et dans le context y'a une petite phrase sur, justement, le contexte conditionnel (ex: "when user is not authenticated").

Et dans vitest ils ont viré le context donc c'est pour pas déboussoller ceux qui ont l'habitude de mocha que je fais ça